s2s_vpn_tunnel.tf

```diff
@@ -19,20 +19,15 @@ resource "aws_customer_gateway" "cdm" {
 resource "aws_vpn_connection" "cdm" {
   provider = aws.sharedservicesprovisionaccount
 
-  customer_gateway_id = aws_customer_gateway.cdm.id
-  # These two resources seem to want a /32, which is incorrect.  See this GitHub issue:
-  # https://github.com/hashicorp/terraform-provider-aws/issues/16879
-  #
-  # This doesn't really matter (I hope), since I will define what
-  # traffic flows into the customer gateway via TGW routing tables.
-  #
-  # We should be able to make use of these parameters once this pull
-  # request is approved and merged:
-  # https://github.com/hashicorp/terraform-provider-aws/pull/17573
-  #
-  # local_ipv4_network_cidr = var.cdm_cidr
-  # remote_ipv4_network_cidr = data.terraform_remote_state.networking.outputs.vpc.cidr_block
-  static_routes_only = true
+  customer_gateway_id     = aws_customer_gateway.cdm.id
+  local_ipv4_network_cidr = var.cdm_cidr
+  # The cidrsubnets() here is because we want to give CDM as small a
+  # subnet as possible, since they are assuming there are no
+  # duplicates across the CISA enterprise.  All the instances that
+  # interest them (the OpenVPN and FreeIPA instances) are in the
+  # bottom /4 of the SharedServices CIDR block.
+  remote_ipv4_network_cidr = cidrsubnets(data.terraform_remote_state.networking.outputs.vpc.cidr_block, 4)[0]
```

Comment:
Magic `4`.

Reply:
The way this is phrased (and coded) makes it sound like the value of `data.terraform_remote_state.networking.outputs.vpc.cidr_block` is a `/24` and you're subnetting it down to a `/28` to get a resultant subnet for the bottom `/4`. If that's correct I think it would be beneficial to elaborate what is being done more in the comments so it is clearer what is happening as I believe that value is set by a variable in [cisagov/cool-sharedservices-networking/vpc.tf](https://github.com/cisagov/cool-sharedservices-networking/blob/5a849d960d25fdf1cfab7a2788023f7a58d14420/vpc.tf#L17). If what I'm saying is wrong then I would like an explanation even more.